packages/cli/test/util/load-plan.test.ts

```diff
@@ -48,7 +48,7 @@ test.serial('expression: load a plan from an expression.js', async (t) => {
     plan: {},
   };
 
-  const plan = await loadPlan(opts as Opts, logger);
+  const plan = await loadPlan(opts as unknown as Opts, logger);
```

Comment:
This typing is pretty awkward. I think you can just remove the explicit casting entirely?
```
const plan = await loadPlan(opts, logger);
```